pkg/cli/approve.go

```diff
@@ -11,14 +11,14 @@ import (
 )
 
 var approveEventCmd = &cobra.Command{
-	Use:     "event [CANVAS_ID] [STAGE_ID] [EVENT_ID]",
+	Use:     "event [CANVAS_ID_OR_NAME] [STAGE_ID] [EVENT_ID]",
```

Comment:
I think it would be better to have the canvas ID or name and stage ID or name as flags instead of parameters here, kinda like the [semaphoreci/cli](https://github.com/semaphoreci/cli) does with `--project-id` and `--project-name` flags. What do you think?